test/run-drun/idx.mo

```diff
@@ -0,0 +1,32 @@
+import CO4 "ok/idx-class"
```

Comment:
Why is this a `run-drun` test? If there is no Internet Computer required, I would suggest to make tests `run` tests whenever possible.

Reply:
Will try to move. Where should I put dependency modules? Currently I stash it under `ok/`, but that looks like an abomination.

Reply:
Ah, when in `run/` don't need a module any more, I can inline the class instead!

Reply:
Why did you need  a module in the first place? You can have inline classes just fine

Reply:
That's what I mean. Remove the module wrapping.

Reply:
ad403730f